java/src/main/java/ai/rapids/cudf/ColumnView.java

```diff
@@ -341,6 +341,16 @@ private enum StripType {
     StripType(int nativeId) { this.nativeId = nativeId; }
   }
 
+  public enum DuplicateKeepOption {
```

Comment:
I would leave a comment (or an `@see`) to mention `cudf::duplicate_keep_option` in `include/stream_compaction.hpp`.  It is vital that the ordinal values are kept in sync.